admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1075,6 +1075,40 @@ $product-page-padding-bottom: 80px !default;
       max-width: 415px;
     }
   }
+
+  #product_description_related_products {
+    .search-with-icon {
+      max-width: 377px;
+    }
+
+    .entities-list-container {
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      margin: 0.5rem -0.5rem;
+    }
+
+    .related-product {
+      display: flex;
+      flex-direction: column;
+      width: 180px;
+      margin: 0.5rem;
+
+      &-image {
+        width: 100%;
+
+        img {
+          display: block;
+          width: auto;
+          max-width: 170px;
+          height: auto;
+          max-height: 170px;
```

Comment:
Depending of the image ratio you choose this may cause some UI issues, better to use the `.img-fluid` class or use the trick I sent before (which is the same as img-fluid) and don't set the height I think so

Reply:
Ok that makes sense, thanks!! I'd prefer to use `img-fluid` then but I'm not sure exactly how. I mean I add the `img-fluid` to the image itself, but then where do I limit the width of my blocks?

Do I keep the `width: 180px;` on `.related-product` ? Or as you say use `max-width: 180px;`? Should I have a min-wdth as well? And finally, how do I handled the "margin" on the image, I set 170px so that there is a gap, should I rely on a padding on `.related-product` maybe?